wordpress-mobile/WordPress-iOS
# Creates a reusable modal bottom drawer sheet

Fixes #13776

To test:

1. Tap the My Sites tab
2. Tap the + button
3. Tap the Blog Post option
4. Tap Publish
5. Swipe up to transition to the expanded mode
6. Swipe down to transition to the collapsed mode
7. Tap the Tags option
8. The view should expand when the keyboard is displayed
9. You should be able to scroll down to the bottom of the view
Note: The tags view has a bug where you can't see the entire last option
10. Tap back and the view should automatically collapse
11. Swipe down to dismiss the view
12. Tap Publish again
13. Tap the dark background view to dismiss the view

PR submission checklist:

- [] I have considered adding unit tests where possible.
